src/ImpactX.cpp

```diff
@@ -100,5 +100,10 @@ namespace impactx
                                       diagnostics::OutputType::PrintParticles,
                                       "diags/output_beam.txt");
 
+        // print the two invariants H and I
+        diagnostics::DiagnosticOutput(*m_particle_container,
+                                      diagnostics::OutputType::PrintInvariants,
+                                      "diags/output_invariants.txt");
```

Comment:
Idea if we add more invariants later
```suggestion
                                      "diags/output_nonlinear_lens_invariants.txt");
```